src/extensions/layout-selector/editor.scss

```diff
@@ -33,29 +33,54 @@
 }
 
 .coblocks-layout-selector-modal {
+	background-color: #f5f5f5;
```

Comment:
Use the proper base-styles variable here.

Reply:
@richtabor, Is there a proper base-style variable to use instead? Looking at `node_modules/@wordpress/base-styles/_colors.scss` I do not see that hex value. I suppose the color would be altered otherwise. Thoughts?

Reply:
Use $gray-100 for now

Reply:
$gray-100